src/DynamoCore/Graph/Workspaces/HomeWorkspaceModel.cs

```diff
@@ -31,6 +31,7 @@ public class HomeWorkspaceModel : WorkspaceModel
         private PulseMaker pulseMaker;
         private readonly bool verboseLogging;
         private bool graphExecuted;
+        private bool executingTask;
```

Comment:
Would you add some comments here even if it is private?

Reply:
Yes will add some comments.